framework/midi/midimodule.cpp

```diff
@@ -42,20 +43,28 @@ using namespace mu::midi;
 
 #ifdef Q_OS_LINUX
 #include "internal/platform/lin/alsamidioutport.h"
+#include "internal/platform/lin/alsamidiinport.h"
 static std::shared_ptr<IMidiOutPort> midiOutPort = std::make_shared<AlsaMidiOutPort>();
+static std::shared_ptr<IMidiInPort> midiInPort = std::make_shared<AlsaMidiInPort>();
 #endif
 
 #ifdef Q_OS_WIN
 #include "internal/platform/win/winmidioutport.h"
+#include "internal/platform/win/winmidiinport.h"
 static std::shared_ptr<IMidiOutPort> midiOutPort = std::make_shared<WinMidiOutPort>();
+static std::shared_ptr<IMidiInPort> midiInPort = std::make_shared<WinMidiInPort>();
 #endif
 
 #ifdef Q_OS_MACOS
 //#include "internal/platform/osx/coremidioutport.h"
+//#include "internal/platform/osx/coremidiinport.h"
 //static std::shared_ptr<IMidiOutPort> midiOutPort = std::make_shared<CoreMidiOutPort>();
+//static std::shared_ptr<IMidiInPort> midiInPort = std::make_shared<CoreMidiInPort>();
 
 #include "internal/dummymidioutport.h"
+#include "internal/dummymidiinport.h"
 static std::shared_ptr<IMidiOutPort> midiOutPort = std::make_shared<DummyMidiOutPort>();
+static std::shared_ptr<IMidiOutPort> midiOutPort = std::make_shared<DummyMidiInPort>();
```

Comment:
line 67 duplicates 66

Reply:
Strange that this even passes the compiler

Reply:
ups 
This is code only for MacOS. 
I didn't compile for MacOS, I asked @psmokotnin  check compilation and fix it

Reply:
fixed